src/Dfe.PlanTech.Application/Content/Queries/GetContentSupportPageQuery.cs

```diff
@@ -0,0 +1,47 @@
+using Dfe.PlanTech.Application.Caching.Interfaces;
+using Dfe.PlanTech.Application.Exceptions;
+using Dfe.PlanTech.Application.Persistence.Interfaces;
+using Dfe.PlanTech.Application.Persistence.Models;
+using Dfe.PlanTech.Domain.Content.Models.ContentSupport;
+using Dfe.PlanTech.Domain.Content.Models.Options;
+using Dfe.PlanTech.Domain.Content.Queries;
+using Dfe.PlanTech.Infrastructure.Application.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Dfe.PlanTech.Application.Content.Queries;
+
+public class GetContentSupportPageQuery : IGetContentSupportPageQuery
+{
+    private readonly ICmsCache _cache;
+    private readonly IContentRepository _repository;
+    private readonly ILogger<GetContentSupportPageQuery> _logger;
+    private readonly GetPageFromContentfulOptions _options;
+
+    public GetContentSupportPageQuery(ICmsCache cache, IContentRepository repository, ILogger<GetContentSupportPageQuery> logger, GetPageFromContentfulOptions options)
```

Comment:
Any reason you don't use primary constructor?

Reply:
Just habit I think. I've changed it though to be more concise [a7ee237](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/a7ee23746b4af6efd015645300904b889a4e35c8)